pkg/meta/redis.go

```diff
@@ -114,6 +114,18 @@ func newRedisMeta(driver, addr string, conf *Config) (Meta, error) {
 		opt.Password = os.Getenv("META_PASSWORD")
 	}
 	opt.MaxRetries = conf.Retries
+	if opt.MinRetryBackoff == 0 {
+		opt.MinRetryBackoff = time.Millisecond * 20
```

Comment:
Currently the default values of two backoffs are different in sentinel mode & non-sentinel mode

Reply:
Also struct `queryMap` can be deleted if it's no longer used.

Reply:
Also `queryMap` can be deleted if it's no longer used.